pkg/operator/terminationobserver/termination_metrics.go

```diff
@@ -0,0 +1,65 @@
+package terminationobserver
```

Comment:
rename file to metrics.go

Reply:
why? i want to be able to grep logs from this file... klog messages from `metrics.go` are not very useful